README.md

```diff
@@ -1,2 +1,81 @@
+
+## Usage
+
+This container can create backups on a [Microsoft SQL Server] container.
+
+**NOTE:**
+The backup is written to a directory `/backup` inside the [Microsoft SQL Server] container, not to a volume in the backup container.
+
+### Tags
+
+| Tag    | Description                                                                             | Size                                                                                                                  |
+|--------|-----------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| latest | Latest master build                                                                     | ![Size](https://shields.beevelop.com/docker/image/image-size/bbtsoftwareag/mssql-backup/latest.svg?style=flat-square) |
+
+### Configuration
+
+These environment variables are supported:
+
+| Environment variable | Default value | Description                                                                                                                                                                                                |
+|----------------------|---------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| DB_SERVER            | `mssql`       | Name or address of the database server to backup. Normally this should be the name of the [Microsoft SQL Server] service.                                                                                  |
+| DB_USER              | `SA`          | User used to connect to the database server.                                                                                                                                                               |
+| DB_PASSWORD          |               | Password used to connect to the database server.                                                                                                                                                           |
+| DB_NAMES             |               | Names of the databases for which a backup should be created.                                                                                                                                               |
+| TZ                   |               | Timezone to use.                                                                                                                                                                                           |
+| CRON_SCHEDULE        | `0 1 * * sun` | Cron schedule for running backups. NOTE: There is no check if there's already a backup running when starting the backup job. Therefore time interval needs to be longer than maximum expected backup time. |
```

Comment:
I would add the information that the time for all databases. e.g. `...than the maximum expected backup time for all databases.`